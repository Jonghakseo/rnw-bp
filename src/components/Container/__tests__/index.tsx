import React from "react";
import { Container } from "..";
import renderer from "react-test-renderer";
import { Text } from "react-native";
import withProvider from "../../../testHelper";

it("should render correctly", () => {
  const component = renderer.create(
    withProvider(
      <Container testID="wrapper">
        <Text>hello</Text>
      </Container>
    )
  );

  expect(component.toJSON()).toMatchSnapshot();
});
