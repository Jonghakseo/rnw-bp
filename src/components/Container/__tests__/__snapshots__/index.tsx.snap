// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#d47271",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
  testID="wrapper"
>
  <Text>
    hello
  </Text>
</View>
`;
